src/test/java/com/criteo/vips/VipsImageTest.java

```diff
@@ -156,6 +157,8 @@ public void TestWriteFromByteArrayShouldNotThrows(@FromDataPoints("filenames") S
         }
     }
 
+    // Ignoring because it crashes Surefire on macOS.
```

Comment:
FAR: we can probably ignore these tests only for MacOS?

Reply:
I checked and wasn't able to find an easy way to do that in Java (except with a big `if (macOS)` around the code, which would mark the test as passing).

Reply:
You can have a look [here](https://github.com/criteo/JVips/blob/master/src/main/java/com/criteo/vips/Vips.java#L80):
`    private static boolean isWindows() {
        return SystemName.indexOf("win") >= 0;
    }`

Reply:
Yes, that's what I'm saying. It's possible to skip it in the code and make it pass, but reporting as "ignored" is non-trivial.

Reply:
Apprently, it's doable with JUnit 5: https://stackoverflow.com/a/49526738/107028. But we currently use JUnit 4, so FAR.